.github/workflows/nightly.yml

```diff
@@ -43,20 +54,27 @@ jobs:
         run: |
           module load xilinx/vivado
           bazel_tests="$(mktemp)"
-          ./bazelisk.sh query 'attr("tags", "[\[ ]cw310_sival[,\]]", tests(//...))' \
+          ./bazelisk.sh query 'attr("tags", "[\[ ]cw310_sival[,\]]", tests(//sw/device/...))' \
                                                     | grep -v examples \
-                                                    | grep -v third_party \
                                                     | grep -v penetrationtests \
                                                     > "$bazel_tests"
-          ./bazelisk.sh test --build_tests_only --target_pattern_file="$bazel_tests"
+          ./bazelisk.sh test --build_tests_only --target_pattern_file="$bazel_tests" || true
```

Comment:
There's a continue-on-error option for the step if you want to prevent a step failure to stop next step running.

Alternatively, make the next step `success() || failure()` so the whole run reports whether any job failed.